pkg/controllers/workgenerator/controller.go

```diff
@@ -530,7 +533,36 @@ func buildAllWorkAppliedCondition(works map[string]*fleetv1beta1.Work, binding *
 		Status:             metav1.ConditionFalse,
 		Type:               string(fleetv1beta1.ResourceBindingApplied),
 		Reason:             workNotAppliedReason,
-		Message:            "not all corresponding work objects are applied",
+		Message:            fmt.Sprintf("work object %s is not applied", notAppliedWork),
```

Comment:
```suggestion
		Message:            fmt.Sprintf("Work object %s is not applied", notAppliedWork),
```
message begins with upper case?